Public/Src/Cache/ContentStore/Vfs/VfsCasConfiguration.cs

```diff
@@ -42,6 +42,26 @@ public class VfsCasConfiguration
         /// </summary>
         public AbsolutePath VfsRootPath { get; }

```

Comment:
>VfsRootPath [](start = 28, length = 11)

Is this still used?

Reply:
Yes, lots of times.

---
In reply to: [298309708](https://github.com/microsoft/BuildXL/pull/519#discussion_r298309708) [](ancestors = 298309708)